### PR TITLE
chore(eslint): use eslint-config-callstack-io

### PR DESCRIPTION
**Pull request by thymikee**

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -141,16 +141,12 @@ class Button extends React.Component<Props, State> {
...
-        if (buttonColor) {
-          backgroundColor = buttonColor;
-        } else {
-          if (primary) {
-            backgroundColor = colors.primary;
-          } else {
-            backgroundColor = theme.dark ? '#535354' : white;
-          }
-        }
+        backgroundColor = theme.dark ? '#535354' : white;
```
Comment (thymikee) [inline src/components/Button.js]: This was autofixed by eslint

File: .eslintrc [hunk 2]
```diff
@@ -1,148 +1,30 @@
...
-    "react/jsx-key": "error",
-    "react/jsx-no-bind": "off",
-    "react/jsx-no-duplicate-props": "error",
-    "react/jsx-no-literals": "off",
-    "react/jsx-no-undef": "error",
-    "react/jsx-pascal-case": "off",
-    "react/jsx-sort-props": "off",
-    "react/jsx-uses-react": "error",
-    "react/jsx-uses-vars": "error",
+    "import/first": "off",
```
Comment (satya164) [inline .eslintrc]: Do we need to disable this rule? I'd prefer if we didn't.

File: docs/pages/src/Home.js [hunk 3]
```diff
@@ -131,28 +131,35 @@ export default class Home extends React.Component<{}> {
...
               </a>{' '}
               to try it out
             </div>
             <a
               href="https://expo.io/@satya164/react-native-paper-example"
               target="_blank"
+              rel="noopener noreferrer"
             >
-              <img {...styles(qrcode)} src="images/expo-qr.png" />
+              <img {...styles(qrcode)} src="images/expo-qr.png" alt="" />
```
Comment (satya164) [inline docs/pages/src/Home.js]: Can you add the alt tag as "Expo QR Code"?

File: .eslintrc (hunk 2 above)
Comment (thymikee) [reply .eslintrc]: Yea, because we would need to mix type imports with regular ones

**Pull request merged**